participantes/yMarceloMaia/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: "3.5"
+
+services:
+  api01: &api
+    image: zanfranceschi/rinha-2024q1-crebito
```

Comment:
vc precisa usar a sua imagem

Reply:
Até achei estranho que tava fácil demais pra ser verdade kk, obrigado pelo toque, tudo isso é novo pra mim.
Estou tentando fazer do jeito certo agora, mas estou quebrando a cabeça com o nginx, está acontecendo o seguinte:

2024/02/11 02:43:40 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://127.0.0.1:80/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/11 02:43:40 [error] 29#29: *1 no live upstreams while connecting to upstream, client: 192.168.96.1, server: , request: "GET /clientes/6/extrato HTTP/1.1", upstream: "http://localhost/clientes/6/extrato", host: "localhost:9999", referrer: "http://localhost:9999/clientes/1/extrato"

Alguma dica?

Reply:
provavelmente, você está usando o docker-compose em modo bridge (que faz sentido), e por isso o nginx não acha o localhost, você precisa configurar o nginx para apontar para os nomes das APIs.

[Essa configuração mostra esse exemplo.](https://github.com/yMarceloMaia/rinha-de-backend-2024-q1/blob/rinha-backend-marcelo/participantes/yMarceloMaia/nginx.conf)

[Note que é o nome definido no docker-compose.](https://github.com/yMarceloMaia/rinha-de-backend-2024-q1/blob/rinha-backend-marcelo/participantes/yMarceloMaia/docker-compose.yml#L6)